### PR TITLE
Increase trimmedErrMessageLen

### PR DESCRIPTION
**Pull request by pradithya**
# TL;DR
Increase `trimmedErrMessageLen` so that error message in list of workflow execution is sufficiently informative.

## Type
 - [X] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [X] Code completed

## Complete description

The current value of `trimmedErrMessageLen` limit the error message shown in the workflow execution list page too much that it become uninformative. Users have to go to the corresponding execution details to get the full error message even for a short error message.
This PR increases the limit of error message length to 10240 so that most error message can be shown properly. 

https://user-images.githubusercontent.com/4023015/236148171-664ef81e-2525-4793-b0b1-8c18d0f7dab3.mov

## Follow-up issue
_NA_

### CODE REVIEW

**Review by pmahindrakar-oss — APPROVED** [review]
The message length limitation was added as part of this PR https://github.com/flyteorg/flyteadmin/pull/523 due to some ListExecution API's reaching the max grpc message limit. 
Max grpc limit size is 4MB, with that size even considering just this lenght of the error message , the List Api can return around 400 executions in the listApi but given that per page limit is 100 this should still be ok. More preferable IMO but would be separate trimmedErrMessageLen for list API calls

**Review by wild-endeavor — COMMENTED** [review]
on second thought @pradithya - can you make this configurable?  your new value as the default is fine, but it would be nice to be configurable.

**Comment by pradithya** [comment]
> on second thought @pradithya - can you make this configurable? your new value as the default is fine, but it would be nice to be configurable.

Make sense, let me have a look.

**Comment by pradithya** [comment]
@wild-endeavor I updated the PR to make it configurable. But it seems the CI is failing due to Github action rate limit. Can you help retry it?

**Review by pingsutw — APPROVED** [review]
@pradithya Thanks, LGTM. Could you resolve branch conflict?
